app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, new_file = command.split()
```

Comment:
Before unpacking you need to validate command have exactly 3 parameters
Also name cp, file, new_file is not clear names
cp -> input_command
file -> source_file_path
new_file -> destination_file_path

Reply:
got it